community/sinit/files/sinit-launch-services.boot

```diff
@@ -0,0 +1,15 @@
+# we are going to start runit and getty here
+
+sinit_run_getty() {
+    for getty in 1 2 3 4 5 6; do
+        while :; do /sbin/getty 38400 tty${getty} 2>&1 ; done &  # busybox getty
+        # while :; do /sbin/getty /dev/tty${getty} linux 2>&1 ; done &  # ubase getty
```

Comment:
Is there not a standard set of `getty` arguments which work regardless of busybox/ubase/etc getty?

Reply:
Nope, there is no standard regarding getty arguments.

Reply:
They are really incompatible, with ubase you must

* NOT specify BAUD_RATE
* Specify full path of tty
* Specify terminal type (linux)

With busybox you must

* Specify BAUD_RATE
* NOT specify full path
* Terminal type is optional

Reply:
This *should* work for both: `getty /dev/tty3 0 linux` (example).

Reply:
This does not work for ubase, by the way.